### ISSUE TITLE
Suggestion: Option to disable baritone's automatic movement

### ISSUE THREAD

**Issue by jvyden420** [cabaletta/baritone#431]
Baritone can be unreliable when moving sometimes, and that can be frustrating. It would be much preferred if baritone would just create a path for the user to follow, while not automatically attempting to follow said path.

**Comment by EmeraldSnorlax** [comment]
this. especially when i want to travel on a pig / across an ocean etc

**Comment by SylveonBottle** [comment]
Could possibly be similar to #326?
Have it so the user can manually follow the line at any point by pressing WASD to override the bot.

**Comment by BitSolve** [comment]
Left clicking already does the job according to #326. Unless WASD is what is something you need, this issue should be closed.

**Comment by leijurv** [comment]
No, that cancels the whole path, this issue is asking for the path to be present and updated live, just not acted upon at all

**Comment by BitSolve** [comment]
Oh, I see. Nvm then

**Comment by user670** [comment]
I would like this to be a thing too. While Baritone tends to avoid creepers, it doesn't care about some other potentially troublesome mobs - more specifically, skeletons with their ranged attacks; it also doesn't place down torches, leaving the path dangerous (should the player return to the place again). I'd like it to be possible to disable auto movement and handle the skeletons by hand.

**Comment by jvyden420** [comment]
UPDATE: You can actually get this temporarily by doing #pause, but the path doesn't keep generating. Sad.
Bump, this feature could be really useful.

**Comment by circl-lastname** [comment]
You can do this i you have impact with macros.

1. Pause
2. Go to Impact
3. Macros
4. Add
4. In the first textbox type `#resume`
5. A second textbox will appear. Type `#pause` in it.
6. Choose the key to bind it to at the bottom
7. Add
8. Press the bind before starting to path

Now press the bind while walking it to and it should generate the path.

**Comment by jvyden** [comment]
been almost a year, i still dont know of any solution

bump

**Comment by l1ving** [comment]
Don't do that, it sends a notification to a ton of people for no good reason

**Comment by SylveonBottle** [comment]
bump

**Comment by TheAlphaReturns** [comment]
bump

**Comment by Staninna** [comment]
i really need this feature

**Comment by lebi376** [comment]
Maybe allow for a hotkey to recalculate a path that stays the same until you press the hotkey again to recalculate it

**Comment by Gaming1010** [comment]
bump?

**Comment by D-a-r-n-o-l-d** [comment]
necro bump

**Comment by gituser11052** [comment]
this would be great for what i'm doing
